draft-ietf-jsonpath-base.md

```diff
@@ -1229,6 +1230,34 @@ Using these operators to compare other types of values produces a "false" compar
 
 The semantics of regular expressions are as defined in {{-iregexp}}.
 
+The logical AND, OR, and NOT operators have the normal semantics of Boolean algebra and
+consequently obey these laws (where `P`, `Q`, and `R` are any expressions with syntax
+`logical-and-expr`, `T` is any true expression, such as `1 == 1`, and `F` is any false expression,
+such as `1 == 0`):
+
+| Law | Expression | Equivalent expression |
+|:--:|:--:|:--:|
+| Associativity of OR | `P || (Q || R)` | `(P || Q) || R`|
+| Associativity of AND | `P && (Q && R)` | `(P && Q) && R`|
+| Commutativity of OR | `P || Q` | `Q || R` |
+| Commutativity of AND | `P && Q` | `Q && R` |
+| Distributivity of OR over AND | `P || (Q && R)` | `(P || Q) && (P || R)`|
+| Distributivity of AND over OR | `P && (Q || R)` | `(P && Q) || (P && R)`|
+| Identity for OR | `P || F` | `P`|
+| Identity for AND | `P && T` | `P`|
+| Anihilator for OR | `P || T` | `T`|
+| Anihilator for AND | `P && F` | `F`|
+| Idempotency of OR | `P || P` | `P`|
```

Comment:
Idempotence is the more usual form.

Reply:
Adopted by https://github.com/ietf-wg-jsonpath/draft-ietf-jsonpath-base/pull/211